test/creds_test.go

```diff
@@ -463,22 +477,39 @@ func (s) TestCredsHandshakeServerNameAuthority(t *testing.T) {
 		t.Fatal(err)
 	}
 	cred := &authorityCheckCreds{}
-	s := grpc.NewServer()
-	go s.Serve(lis)
-	defer s.Stop()
```

Comment:
Why are these changes needed?

Reply:
removing redundant server and using stubserver instead of testservice

Reply:
reverted the changes

Reply:
Thanks, that seems to have fixed the tests.